### PR TITLE
PAL v2.0

### PR DESCRIPTION
**Pull request by zfox23**
DNM for now. PR exists so our team can track progress.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: e93391829be5b21070eb9c497cca324e6296fd9d available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-e933918.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-e933918.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: a81cfc5ac50949ff799de8f462e05c6b29757867 available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-a81cfc5.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-a81cfc5.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0bb676ffc6278a1ca47143190c740d209f03290f available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-0bb676f.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-0bb676f.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e05a9686d9369cbfb989b31d54c5ff53486308de available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-e05a968.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-e05a968.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 35d8afde50bbcc38c29e56e6eb359952dd3605a4 available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-35d8afd.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-35d8afd.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2c43593fe6b29dc6a6f16ce6881fdd14a39fb9c0 available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-2c43593.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-2c43593.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5e3866dab010397122e56196f1310be16d9b3e18 available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-5e3866d.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-5e3866d.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 46d45cdac205b66f3a04e692187fde77a9c7ba90 available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-46d45cd.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-46d45cd.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 513bf7c32d31b9de9252260b1bb98cb5f985cee4 available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-513bf7c.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-513bf7c.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0fb445c58aedf260d9e138100cb764b10be1d6ee available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-0fb445c.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-0fb445c.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 31e9ab6d8797bb750bb8ede6691184fca953d20e available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-31e9ab6.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-31e9ab6.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 783f52dc35a0b5e871d60454e74bec6846640d96 available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-783f52d.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-783f52d.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 19fbe621c06226213d4659caf3040dd4386dace7 available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-19fbe62.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-19fbe62.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: ab9c7b54703a02a5f3d56267374d09452d3cfe9b available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-ab9c7b5.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-ab9c7b5.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9483658d21cd61368bb36a634f5d121c4e810af6 available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-9483658.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-9483658.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 629f366b8637377b954ce7d517dc91eb17456630 available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-629f366.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-629f366.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: a99bcd7a605704da170cf63f9b98f9eb76184709 available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-a99bcd7.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-a99bcd7.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3857eb460ec721f4b1244770323f1adfd368bd7e available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-3857eb4.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-3857eb4.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1db8f59694633aa77986559b9cb893ea4449e07e available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-1db8f59.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-1db8f59.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1045aa9d386f0e0241a231dd7128d047344049e5 available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-1045aa9.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-1045aa9.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 276a2c10e995acd4b174ea22850fc9e3385eaeea available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-276a2c1.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-276a2c1.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7e74017aca3e53d0eff48503a503b7635a15684b available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-7e74017.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-7e74017.dmg

File: interface/resources/qml/hifi/NameCard.qml [hunk 1]
```diff
@@ -76,19 +77,40 @@ Item {
...
         }
+        MouseArea {
+            id: avatarImageMouseArea;
+            anchors.fill: parent
+            enabled: selected || isMyCard;
+            hoverEnabled: true
+            onClicked: {
+            /*
+            THIS WILL OPEN THE BROWSER TO THE USER'S INFO PAGE!
+            I've no idea how to do this yet..
```
Comment (howard-stearns) [inline interface/resources/qml/hifi/NameCard.qml]: See storyCardFrame item within AddressBarDialog.qml.
To see how to activate it, just
            storyCardHTML.url = <<in this case, http://highfidelity.com/user/ + username>>
            storyCardFrame.shown = true;

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4c15d63b4c859397738730dd84f17c225288ce24 available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-4c15d63.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-4c15d63.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 10ced54dab5a268cd81cb28b8975b2ca62cfe014 available at: https://gist.github.com/729712e80c5e5950ea774d3f8070199b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-10ced54.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9843-10ced54.dmg

**Comment by zfox23** [comment]
Deprecated PR - opening new one from updated branch momentarily.

**Pull request closed without merging**